src/main/java/com/wojtek/parkingmeter/helpers/ChargeCalculator.java

```diff
@@ -1,4 +1,67 @@
 package com.wojtek.parkingmeter.helpers;
 
+import com.wojtek.parkingmeter.helpers.enums.TicketType;
+
+import java.time.Duration;
+
+
 public class ChargeCalculator {
+
+    public static double charge(TicketType ticketType, Duration duration) {
+
+        if (TicketType.REGULAR == ticketType)
+            return regular(Math.abs(duration.toHours()));
+
+        if (TicketType.DISABLED.equals(ticketType))
+            return disabled(Math.abs(duration.toHours()));
+
+        return -1.0;
```

Comment:
wow... i co z taką -1 zrobić?

Reply:
W sumie racja. Od tej metody się wymaga aby coś konkretnego mi policzyła a tu fiknie mi -1 i nie wiadomo co z tym zrobić. Jakoś inaczej bym to rozwiązał.